packages/office-ui-fabric-react/src/components/Slider/Slider.tsx

```diff
@@ -244,11 +273,11 @@ export class Slider extends BaseComponent<ISliderProps, ISliderState> implements
     switch (event.which) {
       case getRTLSafeKeyCode(KeyCodes.left):
       case KeyCodes.down:
```

Comment:
Could we make the behavior consistent for horizontal and vertical. Use `down` key to decrease the value.